### PR TITLE
Adds new food item to celebrate Peacekeeper Borgs

### PR DESCRIPTION
**Pull request by Arthurdentist**
**Bleach Kippers**

bleachkipper

By combining the chemicals you'd use for bleach with carp meat, blue crayons, phazon and a bunch of other crap, you can get a lovely baby blue fish meal.

Eating one will turn you blue.

Here is an actual kipper for reference: http://ilovesushi.com/images/menu/shimahokke.jpg

Sprite courtesy of Skowron

### CODE REVIEW

**Comment by FudgePucker** [comment]
Define turning you blue.
Can you provide a screen shot?
Edit: Also I think that this isn't that game changing over powered to require phazon. Pissing off the valid hungry power gaming department with bombs is probably not a good idea.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -4884,3 +4884,17 @@
...
+	desc = "Baby blue and very fishy."
+	icon_state = "bleachkipper"
+	trash = /obj/item/trash/plate
+	food_flags = FOOD_MEAT
+	volume = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/bleachkipper/New()
+	..()
+	reagents.add_reagent(FISHBLEACH, 1)
+	bitesize = 2
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: this doesn't need to be in New()

Comment (Arthurdentist) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: Bitesize?

**Comment by Arthurdentist** [comment]
blueman
this blue

Hard to get a good 'baby blue' colour because you aren't starting with a white surface but I think this gets the idea across pretty well.

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (Kurfursten) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: yeah, you can just put it in the object definition

**Comment by FudgePucker** [comment]
Nice.
Can you grind the food to get the chemical, and how high the dosage does it take to make people blue?
In my opinion I would prefer getting a bottle of that liquid as a clown over a H.O.N.K

**Comment by Arthurdentist** [comment]
1 unit of fish bleach turns you blue but the only way to get it is to make this food which will also only give you 1 unit of fish bleach.

**Comment by Arthurdentist** [comment]
it has a changelog

**Comment by FudgePucker** [comment]
Can someone remind me how much a sheet of phazon produces in the grinder?

**Comment by Shadowmech88** [comment]
1u per sheet.

**Comment by FudgePucker** [comment]
Well fuck me this is going to be an expensive prank.

**Comment by Kurfursten** [comment]
Should Phazon chemical be brought in line with other sheets which produce 20u/ea?

**Comment by Shadowmech88** [comment]
That's going to affect the number of mutations you can get from X sheets of phazon.

**Comment by Kurfursten** [comment]
That's true. Probably better not then.

**Comment by Shadowmech88** [comment]
The recipe could just be changed to require 1u phazon instead of 10u if we want it to require less sheets.

**Comment by Arthurdentist** [comment]
I don't want it to be easy to make.

Its just an awful joke along the lines of the potent ham.

**Comment by Kurfursten** [comment]
Even one sheet of Phazon is a pretty huge barrier to entry, even more than getting a robot head, for a chef.

**Comment by FudgePucker** [comment]
No really, Phazon is a HUEG barrier to entry. 1u per sheet, the recipe requires 10u.
THAT MEANS TEN, TEN WHOLE FUCKING SHEETS OF THE MOST VALUABLE MATERIAL.

**Comment by SkowronX** [comment]
^ Needs a soup based food called "Roboticist Tears"

**Comment by FudgePucker** [comment]
@SkowronX 
Ten units of every metal type. A note containing the robotists name (you can also write rude comments and racial slurs after it if you want). And a pint of doctors delight.

Should also give text to the robotist whose name was on the paper "You feel deep sadness"

**Comment by Arthurdentist** [comment]
recipe now needs one unit of phazon and three blue crayons instead of one

also this has always had a changelog @9600bauds please apologize for incorrectly labelling this PR thanks

**Comment by Shadowmech88** [comment]
>not using the new futuristic in-body changelog system

**Comment by 9600bauds** [comment]
I didn't see the ancient changelog in the included files when I tagged that

**Comment by Arthurdentist** [comment]
I don't keep up with your daft coder memes

**Comment by Arthurdentist** [comment]
wew fucked this up I hate merge conflicts

**Pull request closed without merging**